tools/commands.js

```diff
@@ -392,6 +392,18 @@ function doRunCommand (options) {
     mobileServer = mobileServer + ":" + parsedMobileServer.port;
   }
 
+  let errorAppPath = files.pathJoin(files.convertToStandardPath(__dirname),
```

Comment:
Great point - there is no reason and I will change it to const.

On Tuesday, August 4, 2015, Sashko Stubailo <notifications@github.com>
wrote:

> In tools/commands.js
> <https://github.com/meteor/meteor/pull/4873#discussion_r36215265>:
>
> > @@ -392,6 +392,18 @@ function doRunCommand (options) {
> >      mobileServer = mobileServer + ":" + parsedMobileServer.port;
> >    }
> >
> > +  let errorAppPath = files.pathJoin(files.convertToStandardPath(__dirname),
>
> Is there any reason this is let and not const?
> https://github.com/meteor/javascript#13.1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/meteor/meteor/pull/4873/files#r36215265>.
>